### DISCUSSION TITLE
how to change the options in accounts

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16996] [How to] [answered]
<img width="398" alt="image" src="https://user-images.githubusercontent.com/18006681/228754987-e48f9a67-fc63-49fb-84e5-8da203a5dfaa.png">

when i click accounts, there are 6 options --  accounts, leads, vendors..... 
how to limit the options of just accounts leads?

**Comment by apcloic** [comment, marked as answer]
@newminder 
Hi, I don't think you can do that from the GUI but you may try to remove unwanted modules directly in vtiger_fieldmodulerel table.
Regards,

**Reply by newminder** [reply]
thank you. with your words,  I changed another filed options.   seems the 'link' field is applied with other rules.
